README.md

```diff
@@ -70,44 +101,39 @@ Cutting edge NodeJS:
 {
     "root": true,
     "extends": [
-        "@moxy/eslint-config/es9",
-        "@moxy/eslint-config/addons/node"
+        "@moxy/eslint-config/node"
     ]
 }
 ```
 
-Cutting edge NodeJS with ES modules (requires a transpiler such as Babel or similar):
+Note that by setting `root` to true, we ensure that no ancestor configuration is used which also improves `ESLint` performance because no more file lookups need to be done.
 
-```json
-{
-    "root": true,
-    "extends": [
-        "@moxy/eslint-config/es9",
-        "@moxy/eslint-config/addons/node",
-        "@moxy/eslint-config/addons/es-modules"
-    ]
-}
-```
 
-Old ES5 in the browser:
+## Development
 
-```json
-{
-    "root": true,
-    "extends": [
-        "@moxy/eslint-config/es5",
-        "@moxy/eslint-config/addons/browser"
-    ]
-}
-```
-
-Note that by setting `root` to true, we ensure that no ancestor configuration is used which also improves `ESLint` performance because no more file lookups need to be done.
+After cloning the repository, install and link the dependencies of all the packages with:
 
+`$ lerna bootstrap`
 
 ## Tests
 
-`$ npm test`   
-`$ npm test -- --watch` during development
+`$ lerna run test`
```

Comment:
Can we use `lerna exect` to into the `test` script to keep `npm t`? See: https://github.com/lerna/lerna/tree/master/commands/exec